docs/connecting-data/connecting-to-apis-and-external-data/stripe-api.md

```diff
@@ -5,5 +5,149 @@ description: How to Connect DataDistillr to the Stripe API
 # Connecting to Stripe
 
 ## First Steps with Stripe
+Set up an account with [Stripe](https://dashboard.stripe.com/register){target=_blank}.
 
-Details coming soon!
\ No newline at end of file
+???+ cost "Cost"
+
+    This is a [paid](https://stripe.com/pricing){target=_blank} API. 
+
+
+???+ rlimit "Rate Limits"
+
+    The Stripe API employs a number of [safeguards](https://stripe.com/docs/rate-limits){target=_blank} against bursts of incoming traffic to help maximize its stability. 
+    Users who send many requests in quick succession may see error responses that show up as status code 429.
+
+## How to Connect DataDistillr to Stripe
+To set up a data source connection for Stripe, you will need to have:
+
+- A unique [name](#name) for your data source connection to be used in queries.
+- The [Secret Key](#secret-key) generated by Stripe.
+
+
+
+### Data Source Form
+
+To locate the Stripe form, follow the steps in [Connecting Your Data to DataDistillr](../../). When you get to the window to choose the data source type, select API as shown below.&#x20;
+
+![Select API from the available choices][image-5]
+
+On the API screen, select Stripe from list of API forms as shown in the image below.
+
+![Select Stripe API from available choices][image-6]
+
+The following form will appear. Instructions can be found below on how to find the information required to fill each field on the Stripe API form.
+
+Once you have filled out all the fields, press the green 'Save' button, and your API will be connected!
+
+![Stripe Form][image-1]
+
+### Name
+
+Enter any name that will help you recognize this data source from within your query window. &#x20;
+
+!!! info "Acceptable characters include"
+
+    * lowercase alphanumeric characters
+    * underscores
+
+### Secret Key
+
+=== "1. Sign In"
+
+    You can access your Secret Key at your [Stripe API Keys Page](https://dashboard.stripe.com/apikeys){target=_blank}. Sign in if necessary. Click "Reveal Key"
```

Comment:
"Reveal Test Key", Is there a way to zoom this in to just see the main API keys section?

Reply:
Modify text to "Reveal Test Key"